src/_includes/navigation-main.html

```diff
@@ -19,8 +19,8 @@
         {%- if page_url_path contains '/*/community/' %} active {%- endif -%}">Community</a>
     </li>
     <li>
-      <a href="https://dartpad.dev" target="_blank" class="nav-link no-automatic-external 
-        {%- if page_url_path contains '/*/#dartpad' %} active {%- endif -%}">Try Dart</a>
+      <a href="/#try-dart" class="nav-link no-automatic-external
```

Comment:
We don't need the `no-automatic-external` here, do we?

Reply:
good catch, removed